### ISSUE TITLE
lint: add commentLeadingSpace checker

### ISSUE THREAD

**Issue by Quasilyte** [go-critic/go-critic#469]
Detects single-line comments without leading space.

**Before:**
```go
//this is a comment 
```
**After:**
```go
// this is a comment
```

<hr>

Note that it should check against comment directives.
This should not trigger a warning:
```go
//foo: this may be a directive for "foo"
```
Simple check for `^\w+:` may suffice.

One can use `VisitLocalComment` method to visit all comment groups.
Make sure to skip `/**/`-style comments.

**Comment by cristaloleg** [comment]
Maybe we should just extend docStub, wdyt?

**Comment by Quasilyte** [comment]
Nope. docStub finds documentation stubs that silence go lint.
Unrelated.

**Comment by Quasilyte** [comment]
Proposed checker name seems too lengthy. If we can find a better one, it would be great.

**Comment by cristaloleg** [comment]
`spacedComment` ?

**Comment by Quasilyte** [comment]
Hm, but we're looking for comments without space. :sweat_smile:

**Comment by cristaloleg** [comment]
`un-` ;D
